templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,123 @@
+            ...
+          ]
+          links: [], (array) (default: []): List of links (required if Card type is a 'tile'). Format: [
+            {
+              label: (string) (default: ''): Label of link
+              path: (string) (default: ''): Link url (href attribute)
+            },
+            ...
+          ]
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _content_item = {
+  type: 'default',
+  description: '',
+  meta: [],
+  title: {},
+  image: '',
+  image_position: 'before',
+  infos: [],
+  tags: [],
+  links: []
+} %}
+
+{% if content_item is defined %}
+  {% set _content_item = _content_item|merge(content_item) %}
+{% endif %}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<article class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  {% if _content_item.image and _content_item.image_position == 'before' %}
+    <div role="img" aria-label="{{ _content_item.image.alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-lg-block"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:5rem;width:7.5rem"></div>
+    <div role="img" aria-label="{{ _content_item.image.alt }}"
+         class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-none ecl-u-d-lg-none"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:8.75rem;width:13.125rem">
+    </div>
+  {% endif %}
+  <div class="ecl-u-flex-grow-1">
+    {% if _content_item.meta is not empty %}
+      <div class="ecl-content-item__meta ecl-u-type-s ecl-u-type-color-grey-75">{{ _content_item.meta|join(" | ") }}</div>
+    {% endif %}
+    <div class="ecl-content-item__title ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+      {%- if _content_item.title.path %}
+        {% include '../ec-component-link/link.html.twig' with {
+          link: _content_item.title
+        } only %}
+      {% else %}
+        {{- _content_item.title.label }}
+      {%- endif -%}
+    </div>
+    <div class="ecl-content-item__description ecl-u-type-paragraph ecl-u-type-color-grey-100 ecl-u-mt-xs">{{ _content_item.description }}</div>
```

Comment:
This is a `p` tag [on the playground](https://ec.europa.eu/component-library/playground/ec/?path=/story/templates-compositions--content-items).

Reply:
We have an issue if we want to use a `p` because we are including a <div class='ecl-editor>...</div>' and it breaks the html structure.
For example, if you check the kernel test `ParagraphsTest::testListItem` the result is `<p ...><div class='ecl-editor'><p>...</p></div></p>`.
If we want to use `p` we should change the field description on the paragraph.

Reply:
Ok but I think the issue is the `<div class='ecl-editor'>` getting into the item description right? How that ends up there?

Reply:
The issue here is that the description field on oe_paragraph is a `field_oe_text_long` and not a `field_oe_text`.
field_oe_text_long includes `<p>` and other tag.
According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p
`Paragraphs are block-level elements, and notably will automatically close if another block-level element is parsed before the closing </p> tag.`